src/common/constants.js

```diff
@@ -5,3 +5,7 @@ export const DELETE_BUTTON = "Delete Button";
 export const LOGO = "Logo";
 export const SEARCH_ICON = "Search Icon";
 export const ENTER = "Enter";
+export const TASK_LOGO = "Task Logo";
+export const TASK_PER_PAGE = 9;
+export const LOAD_MORE = "Load More";
+export const SHOW_LESS = "Show Less";
```

Comment:
add some context to the constants.

```suggestion
export const LABEL_LOAD_MORE = "Load More";
export const LABEL_SHOW_LESS = "Show Less";
```